client/dashboard/app/queries/site-users.ts

```diff
@@ -8,5 +10,10 @@ export const siteCurrentUserQuery = ( siteId: number ) => ( {
 export function siteUserDeleteMutation( siteId: number ) {
 	return {
 		mutationFn: ( userId: number ) => deleteSiteUser( siteId, userId ),
+		onSuccess: () => {
+			queryClient.invalidateQueries( siteByIdQuery( siteId ) );
```

Comment:
Hmm.... this pattern is used quite a lot... maybe we should consider renaming it to `siteByIdQueries()` and include `[ 'site', siteId ]` and `[ 'sites' ]` in the predicate. So that we can simply just invalidate this one. 🤔